maze_transformer/evaluation/plot_maze.py

```diff
@@ -116,9 +123,27 @@ def add_multiple_paths(self, path_list: list):
                 self.add_predicted_path(path=path)
         return self
 
-    def show(self) -> None:
+    def add_node_values(
+        self, node_values: NDArray, color_map: str = "Blues"
```

Comment:
it feels like this should be either a parameter to the `__init__` or to `add_node_values`, but not both. Did I misread this when I made my original comment?

Reply:
My logic for the `add_node_values` function is it takes all relevant arguments for node highlighting, including the colormap. However other plotting functions within `MazePlot` class need to access the colormap parameter, thats why I initialize the class variable `self.node_color_map` in the `__init__` function. Should I initialize the `self.node_color_map` variable in another place than inside `__init__` function?

Reply:
this makes sense now! I think the current interface is good.